_includes/footer.html

```diff
@@ -41,7 +41,7 @@ <h6>Knowledge Base</h6>
                 <div class="medium-2 columns end">
                     <h6>OME Public</h6>
                     <ul class="menu vertical">
-                        <li><a href="https://twitter.com/openmicroscopy" target="_blank"><i class="fa fa-fw fa-twitter"></i> Twitter</a></li>
+                        <li><a href="https://bsky.app/profile/openmicroscopy.org" target="_blank"><i class="fa fa-fw fa-twitter"></i> Bluesky</a></li>
```

Comment:
Remove ``fa-twitter`` at least from this line. I do not think we can publicize bluesky link with twitter icon...